sdk/cognitiveservices/azure-resourcemanager-cognitiveservices/src/test/java/com/azure/resourcemanager/cognitiveservices/CognitiveServicesManagerTests.java

```diff
@@ -0,0 +1,108 @@
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.cognitiveservices.models.Account;
+import com.azure.resourcemanager.cognitiveservices.models.AccountProperties;
+import com.azure.resourcemanager.cognitiveservices.models.NetworkRuleAction;
+import com.azure.resourcemanager.cognitiveservices.models.NetworkRuleSet;
+import com.azure.resourcemanager.cognitiveservices.models.PublicNetworkAccess;
+import com.azure.resourcemanager.cognitiveservices.models.Sku;
+import com.azure.resourcemanager.resources.ResourceManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class CognitiveServicesManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST3;
+    private String resourceGroupName = "rg" + randomPadding();
+    private CognitiveServicesManager cognitiveServicesManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        cognitiveServicesManager = CognitiveServicesManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateAccount() {
+        Account account = null;
+        try {
+            String accountName = "account" + randomPadding();
+            // @embedmeStart
+            account = cognitiveServicesManager.accounts()
+                .define(accountName)
+                .withExistingResourceGroup(resourceGroupName)
+                .withRegion(REGION)
+                .withKind("CognitiveServices")
```

Comment:
Does backend really need this?

Reply:
Redundant parameter settings have been removed in the new version.

Reply:
Is `withKind("CognitiveServices")` required?

Reply:
I tried to remove `withKind("CognitiveServices")`, but an exception occurs.
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/30c1a070-895f-4a67-a6e6-00376430e3bd)
So it is required.

Reply:
Thanks. Crazy backend, as if it does not know what service it is...